src/main.cpp

```diff
@@ -75,12 +91,13 @@ void setup()
     #endif
     #ifdef BLUEPILL
     SPI.setMISO(SPI_MISO);
-    SPI.setMOSI(SPI_MOSI);
-    SPI.setSCLK(SPI_SCK);
+	SPI.setMOSI(SPI_MOSI);
+	SPI.setSCLK(SPI_SCK);
     SPI.begin(115200);
     #endif
 
     DW1000Ranging.initCommunication(PIN_RST, PIN_SS, PIN_IRQ); //Reset, CS, IRQ pin
+    //define the sketch as anchor. It will be great to dynamically change the type of module
```

Comment:
I guess this comment can be deleted. We can build the firmware for different type of a module already. See https://github.com/LSyren/UWB-Localization-Exploration/pull/3/files#r1116668569